test/cpp/jit/CMakeLists.txt

```diff
@@ -74,7 +75,7 @@ if(USE_SYSTEM_ONNX)
   target_link_libraries(test_jit PRIVATE onnx_proto onnx)
 endif()
 
-set(JIT_TEST_DEPENDENCIES torch gtest)
+set(JIT_TEST_DEPENDENCIES jitbackend_test gtest)
```

Comment:
On Linux, linker would not add a dependency for shared library, unless target actually depends on symbol from that library.
One way to workaround it is to add something like:
```
+if(LINUX)
  #Update to target_link_options when CMake version can be upgraded
  target_link_libraries(test_jit PRIVATE "-Wl,--no-as-needed,$<TARGET_FILE:jitbackend_test>")
endif()
```

Reply:
Thanks @malfet !